overlord/ifacestate/handlers.go

```diff
@@ -67,8 +67,8 @@ func journalQuotaLayout(quotaGroup *quota.Group) []snap.Layout {
 // getExtraLayouts helper function to dynamically calculate the extra mount layouts for
 // a snap instance. These are the layouts which can change during the lifetime of a snap
 // like for instance mimicking systemd journal namespace mount layouts.
-func getExtraLayouts(st *state.State, snapInstanceName string) ([]snap.Layout, error) {
-	snapOpts, err := servicestate.SnapServiceOptions(st, snapInstanceName, nil)
+func getExtraLayouts(st *state.State, snapInfo *snap.Info) ([]snap.Layout, error) {
+	snapOpts, err := servicestate.SnapServiceOptions(st, snapInfo, nil)
```

Comment:
as the code below is still only snapOpts.QuotaGroup based it means we need more changes to have different journal namespace per services? or we decide not to support setting those in per service quota groups?